usr.sbin/jail/jail.conf.5

```diff
@@ -238,10 +240,19 @@ bar {
 \[char46]include "/usr/local/etc/jail.*.conf";
 .Ed
 .Sh SEE ALSO
-.Xr jail_set 2 ,
```

Comment:
Jail_set(2) seems to be a mlink to jail(2) describing a whole family of jail related syscalls, so use that. Bonus: looks very cohesive next to jail(3).